src/modules/modalMeals.js

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable */
+import commentsCounter from "./commentCount.js";
+
+const modalDetailsContent = document.querySelector(".modal-details-content");
+const url ="https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/";
+const key = "ZbL3vMDmBGqIrLmuwzny";
+
+function mealsFunction(meal) {
```

Comment:
- [ ] Since you use ES6, it would be nicer to use arrow functions instead of the `function` keyword. I recommend refactoring all your functions to arrow functions.